functions/ingest/format/defaultTsdb.go

```diff
@@ -0,0 +1,87 @@
+package format
```

Comment:
Source file names are generally kept lowercase in Go.

Reply:
Also, it's a bit weird to name it for the default format.

Reply:
why? this is indeed the default

Reply:
Probably there's no need to put this in a separate package at all. Could just be `ingest/format.go`

Reply:
you mean all formats in the same file? I prefer it to be a multi package project for the sake of testing the option to build that kind of project (currently its the only one I think)